src/main/java/com/softserve/repository/ICategoryRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.softserve.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.softserve.model.Category;
+
+@Repository
+public interface ICategoryRepository extends JpaRepository<Category, Integer> {
+	//Custom query for searching main categories or parent categories
```

Comment:
for documentation purposes on methods, use the /* */ block instead of singular comment line (//)